setup.py

```diff
@@ -30,6 +30,7 @@
     },
 
     install_requires=[
-        'pylint>=1.4.0',
+        'pylint==1.4.4',
+        'pylint-django==0.6.1',
```

Comment:
There is no correct pattern, there are evolving best practices.  I don't see the advantage of setup.py reading a text file.  I'll rename the file to dev-requirements.txt.  The README will direct them to install from that if they are developing in the repo.  The setup.py will get the correct requirements for installed use.  OK?